extensions/amp-jwplayer/0.1/test/test-amp-jwplayer.js

```diff
@@ -87,59 +87,41 @@ describes.realWin(
         'data-playlist-id': '482jsTAr',
         'data-player-id': 'sDZEo0ea',
         'data-content-search': '__CONTEXTUAL__',
-        'data-content-contextual': true,
       });
       const iframe = jw.querySelector('iframe');
       expect(iframe).to.not.be.null;
       expect(iframe.tagName).to.equal('IFRAME');
       expect(iframe.src).to.equal(
-        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=title_tag&contextual=true'
+        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=title_tag'
       );
     });
     it('renders with a playlist and all parameters', async () => {
       const jw = await getjwplayer({
         'data-playlist-id': '482jsTAr',
         'data-player-id': 'sDZEo0ea',
         'data-content-search': 'dog',
-        'data-content-contextual': true,
-        'data-content-recency': '9D',
         'data-content-backfill': true,
       });
       const iframe = jw.querySelector('iframe');
       expect(iframe).to.not.be.null;
       expect(iframe.tagName).to.equal('IFRAME');
       expect(iframe.src).to.equal(
-        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=dog&contextual=true&recency=9D&backfill=true'
+        'https://content.jwplatform.com/players/482jsTAr-sDZEo0ea.html?search=dog&backfill=true'
       );
     });
     it('renders with a playlist and contextual, backfill parameters', async () => {
```

Comment:
`contextual` has been removed, delete from test descriptions.

Reply:
This test is no longer valid @alanorozco , so I have just removed it.